server/src/main/java/org/elasticsearch/search/SearchModule.java

```diff
@@ -281,13 +281,6 @@ public class SearchModule {
         Setting.Property.NodeScope
     );
 
-    public static final Setting<Boolean> SEARCH_CONCURRENCY_ENABLED = Setting.boolSetting(
-        "search.concurrency_enabled",
-        true,
-        Setting.Property.NodeScope,
-        Setting.Property.Dynamic
-    );
-
```

Comment:
I moved this to SearchService together with the other existing search settings (including minimum docs per slice which affects search concurrency)